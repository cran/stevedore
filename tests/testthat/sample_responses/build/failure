{"stream":"Step 1/5 : FROM alpine:latest"}
{"stream":"\n"}
{"stream":" ---\u003e 3fd9065eaf02\n"}
{"stream":"Step 2/5 : COPY iterate /usr/local/bin/iterate"}
{"stream":"\n"}
{"errorDetail":{"message":"COPY failed: stat /var/lib/docker/tmp/docker-builder712978910/iterate: no such file or directory"},"error":"COPY failed: stat /var/lib/docker/tmp/docker-builder712978910/iterate: no such file or directory"}
