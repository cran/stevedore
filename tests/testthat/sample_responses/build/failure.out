Step 1/5 : FROM alpine:latest
 ---> 3fd9065eaf02
Step 2/5 : COPY iterate /usr/local/bin/iterate
